SeriesMaster-Frontend/App/components/AppInputText.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import styled from "styled-components/native";
+import PropTypes from "prop-types";
+
+const InputContainer = styled.View`
+  background: #fcfcfc;
+  border-radius: 25px;
+  margin: 10px;
+`;
+
+const InputText = styled.TextInput`
+  color: #20232a;
+  padding: 10px;
+  margin-left: 5px;
+`;
+
+const AppInputText = ({ placeholder, query, onChange }) => {
+  return (
+    <InputContainer>
+      <InputText
+        value={query}
+        onChange={() => onChange(event.target.value)}
```

Comment:
`event` is undefined

Reply:
In this case it doesn't seem to make a difference. I can still log the event and it is recognizing it as an input event.